### ISSUE TITLE
ue4 Docker build fails on linux for 4.26.0-preview-2

### ISSUE THREAD

**Issue by meetakshay99** [adamrehn/ue4-docker#102]
Output of the `ue4-docker info` command:

```

ue4-docker version:         0.0.69 (latest available version is 0.0.71)
Operating system:           Linux (Ubuntu 18.04.4 LTS, 5.4.0-1025-aws)
Docker daemon version:      19.03.12
NVIDIA Docker supported:    Yes
Maximum image size:         No limit detected
Available disk space:       180 GiB
Total system memory:        30.91 GiB physical, 0 bytes virtual
Number of processors:       4 physical, 8 logical

```

Command used to build docker image was:
ue4-docker build custom:4_26_Preview_2 -repo=https://github.com/EpicGames/UnrealEngine.git -branch=4.26.0-preview-2 --cuda --no-cache --no-engine -username=My_Git_Username -password=My_Git_Password > 4_26_Preview_2_DockerBuild.log 2>&1

Error part at the end of the build log:

Running: xbuild /home/ue4/UnrealEngine/Engine/Saved/CsTools/Engine/Source/Programs/AutomationTool/Gauntlet/Gauntlet.Automation.csproj /property:Platform=AnyCPU /property:Configuration=Development /verbosity:minimal /nologo /verbosity:quiet /nologo /p:DefineConstants=MONO /p:DefineConstants=__MonoCS__ /p:TargetFrameworkProfile=
/home/ue4/UnrealEngine/Engine/Saved/CsTools/Engine/Source/Programs/AutomationTool/Gauntlet/Gauntlet.Automation.csproj: error : /home/ue4/UnrealEngine/Engine/Saved/CsTools/Engine/Source/Programs/AutomationTool/Gauntlet/Gauntlet.Automation.csproj: /home/ue4/UnrealEngine/Engine/Saved/CsTools/Engine/Source/Programs/AutomationTool/Gauntlet/Gauntlet.Automation.csproj could not import "../../../../Platforms/*/Source/Programs/AutomationTool/Gauntlet/*.Gauntlet.targets"
Took 0.808122s to run xbuild, ExitCode=1
Command failed (Result:1): xbuild /home/ue4/UnrealEngine/Engine/Saved/CsTools/Engine/Source/Programs/AutomationTool/Gauntlet/Gauntlet.Automation.csproj /property:Platform=AnyCPU /property:Configuration=Development /verbosity:minimal /nologo. See logfile for details: 'xbuild-2020.10.15-19.34.34.txt'
AutomationTool exiting with ExitCode=1 (Error_Unknown)
RunUAT ERROR: AutomationTool was unable to run successfully.
The command '/bin/sh -c ./Engine/Build/BatchFiles/RunUAT.sh BuildGraph -target="Make Installed Build Linux" -script=Engine/Build/InstalledEngineBuild.xml -set:HostPlatformOnly=true -set:WithDDC=$BUILD_DDC && rm -R -f /home/ue4/UnrealEngine/LocalBuilds/InstalledDDC && python3 /tmp/exclude-components.py /home/ue4/UnrealEngine/LocalBuilds/Engine/Linux $EXCLUDE_DEBUG $EXCLUDE_TEMPLATES && echo '' && echo 'RUN directive complete. Docker will now commit the filesystem layer to disk.' && echo 'Note that for large filesystem layers this can take quite some time.' && echo 'Performing filesystem layer commit...' && echo ''' returned a non-zero code: 1
[ue4-docker build] Error: failed to build image "adamrehn/ue4-minimal:4_26_preview_2-cudagl9.2".

Full log can be found [here](https://drive.google.com/file/d/1ocN8MZ8CNDwlTOENOzLOJ3ZiL8yZqhhu/view?usp=sharing)

**Comment by TBBle** [comment]
This looks like a UE4 bug fixed in 4.26.0-preview-3, see https://github.com/EpicGames/UnrealEngine/commit/0cfad88ecdccad7321a7b3f08a031f3cf141d10c.

**Comment by adamrehn** [comment]
Closing, since this was an upstream Unreal Engine issue.
